src/Microsoft.AspNet.OData.Shared/Query/DefaultSkipTokenHandler.cs

```diff
@@ -0,0 +1,387 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using System.Linq.Expressions;
+
+using Microsoft.AspNet.OData.Common;
+using Microsoft.AspNet.OData.Formatter;
+using Microsoft.AspNet.OData.Formatter.Serialization;
+using Microsoft.AspNet.OData.Query.Expressions;
+using Microsoft.OData;
+using Microsoft.OData.Edm;
+using Microsoft.OData.UriParser;
+
+namespace Microsoft.AspNet.OData.Query
+{
+    /// <summary>
+    /// Default implementation of SkipTokenHandler for the service. 
+    /// </summary>
+    public class DefaultSkipTokenHandler : SkipTokenHandler
+    {
+        private const char CommaDelimiter = ',';
+        private static char propertyDelimiter = ':';
+        internal static  DefaultSkipTokenHandler Instance = new DefaultSkipTokenHandler();
+
+        /// <summary>
+        /// Constructor for DefaultSkipTokenHandler - Sets the Property Delimiter
+        /// </summary>
+        public DefaultSkipTokenHandler()
+        {
+            IsDeltaFeedSupported = false;
```

Comment:
>IsDeltaFeedSupported [](start = 12, length = 20)

where does it use?

Reply:
It is used in ODataDeltaFeedSerializer. To determine if the implementation should be used for writing the NextLink. 

---
In reply to: [282611153](https://github.com/OData/WebApi/pull/1724#discussion_r282611153) [](ancestors = 282611153)